src/m365/spo/commands/tenant/tenant-appcatalog-add.ts

```diff
@@ -118,7 +118,7 @@ class SpoTenantAppCatalogAddCommand extends SpoCommand {
       logger.logToStderr(`Creating app catalog at ${options.url}...`);
     }
 
-    const siteClassicAddOptions = {
+    const siteAddOptions = {
```

Comment:
@martinlingstuyl sorry but I am having some trouble testing this change locally. 
Frist, thing I noticed is the `owner` option. In the `spo classic add` id is `owner` but in the `spo add` the parameter is actually `owners` 🤔
Second, when testing this I get `Error: Value cannot be null. Parameter name: alias` error. Seems strange as this parameter is optional but maybe for this webTemplate is obligatory. 
May I kindly ask you to double check ? 
BTW sorry for the stupid command last time 😋 I didn't notice it was the `add` command here 😉

Reply:
Will check this out @Adam-it, it can be at I missed something!

Reply:
Okay, it appears I did not test this well enough 😂
There where some things wrong. Aside from that we are still creating a classic App Catalog, while SharePoint does something different these days. It creates a modern app catalog, much faster creating it. We'll need to look into that.